lib/index.js

```diff
@@ -67,6 +67,14 @@ function preprocessor(css, options, filename) {
     return settings ? plugin(settings) : plugin;
   });
 
+  // autoprefixer should always be the last plugin
+  plugins.push(require('autoprefixer')(options.autoprefixer));
```

Comment:
yep agreed moved the `require` to the top.
Regarding `postcss-reporter` shall we agree that we enforce the following order?
```
postcss-easy-import

user defined plugins or defaults

[optionally encapsulation ones if encapsulateStyles is true]
autoprefixer
postcss-reporter
```